src/stk/io/mod.rs

```diff
@@ -0,0 +1,71 @@
+use emdrv::gpioint;
+use {cmu, gpio};
+
+pub struct Button {
+    port: gpio::Port,
+    pin: u8,
+}
+
+impl Button {
+
+    fn new(port: gpio::Port, pin: u8) -> Button {
+        Button {
+            port: port,
+            pin: pin,
+        }
+    }
+
+    pub fn init(&self) {
+        cmu::clock_enable(cmu::Clock::GPIO, true);
+        gpio::pin_mode_set(self.port, self.pin as u32, gpio::Mode::Input, 0);
+        gpio::int_config(self.port, self.pin as u32, false, true, true);
+
+        gpioint::init();
+    }
+
+    pub fn pb0() -> Button {
+        Button::new(gpio::Port::B, 9)
+    }
+
+    pub fn pb1() -> Button {
```

Comment:
In that case we should include both interfaces. As `led0` and friends are a getters that can be use multiple times while the `init` should only be called once.